include/aikido/constraint/DefaultOutcome.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef AIKIDO_CONSTRAINT_DEFAULTOUTCOME_HPP_
+#define AIKIDO_CONSTRAINT_DEFAULTOUTCOME_HPP_
+
+#include "TestableOutcome.hpp"
+
+namespace aikido {
+namespace constraint {
+
+/// Simple default TestableOutcome derivative class. An instance of this class
+/// is returned when createOutcome() is called on an instance of a class that
+/// inherits Testable, but has no corresponding TestableOutcome derivative
+/// implemented.
+class DefaultOutcome : public TestableOutcome
```

Comment:
I suggest renaming this to `DefaultTestableOutcome`, since it inherits from `TestableOutcome` and is currently tied to `Testable` constraints. It's unclear if we'll be able to use the same API on other constraint types.